tools/docker/docker-compose-mac.yml

```diff
@@ -209,13 +210,16 @@ services:
     ports:
       - '${EDA_PROXY_PORT:-3128}:3128'
 
-  eda-ui:
-    image: ${EDA_UI_IMAGE:-quay.io/ansible/eda-ui}:${EDA_UI_VERSION:-main}
+  eda-nginx:
+    image: ${EDA_NGINX_IMAGE:-docker.io/nginx:alpine}
     environment: *common-env
+    command: nginx -g "daemon off;"
     ports:
       - '${EDA_UI_PORT:-8443}:443'
```

Comment:
should we change EDA_UI_PORT to EDA_API_PORT?

Reply:
I didn't it in other places to keep backward compatibility, there is other development stuff that currently relies on this envvar.

Reply:
@bzwei exactly what Alejandro is noting. This also maintains consistency with the changes here - https://github.com/ansible/eda-server/pull/1071/files

Reply:
we can do a later PR that removes other dependencies